### ISSUE TITLE
GitHub logo shows up on the left side. 

### ISSUE THREAD

**Issue by levlaz** [felicianotech/discourse.directory#25]
I am pretty certain this is a bug and is meant to be on the right side. 

MacOS 

Firefox Dev Edition 

![](https://feliciano.tech/assets/img/feature/discourse-directory.png)

**Comment by felicianotech** [comment]
Yes that is. Which version of macOS? Also, does Firefox Dev Edition have a version number? Or is it like a rolling release?

**Comment by felicianotech** [comment]
Whatever causes this issue in the Dev Edition has made it to the FireFox 57 stable release.
